tests/memory_pool_test.cpp

```diff
@@ -0,0 +1,456 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "merlin/memory_pool.hpp"
+#include <cuda_runtime_api.h>
```

Comment:
Change the name to memory_poll_test.cu, and remove the `cuda_runtime_api.h`.

Reply:
Since we switch to `cuh` and with that to NVCC compilation, it is indeed no longer necessary. Done.!